napalm_ios/ios.py

```diff
@@ -947,19 +947,26 @@ def get_interfaces(self):
         interface_dict = {}
         for line in output.splitlines():
 
-            interface_regex = r"^(\S+?)\s+is\s+(.+?),\s+line\s+protocol\s+is\s+(\S+)"
-            if re.search(interface_regex, line):
-                interface_match = re.search(interface_regex, line)
-                interface = interface_match.groups()[0]
-                status = interface_match.groups()[1]
-                protocol = interface_match.groups()[2]
-
-                if 'admin' in status:
-                    is_enabled = False
+            interface_regex_simple = r"^(\S+?)\s+is\s+(.+?)"
```

Comment:
I think matching on the word ' is ' is too general. I am worried that will fail on some set of data.

I think we should just add a special case of the psuedowire.

Reply:
Sorry, I dropped the ball here. I think that's fine for the time being. if this is going to fail on other input data, we can always add another test case. Would you agree?

Reply:
I think that regular expression is too general. It will very likely fail. We should just correct it. So I think we should hold off on this and not include it with the 0.8.0 release.